adb/fastboot.py

```diff
@@ -16,7 +16,12 @@
 import argparse
 import binascii
 import collections
-import io
+import sys
+
+if sys.version_info < (3, 0):
```

Comment:
All right, but there is a difference between `io` from 2.7 and `io` from 3.6

On 2.7 : 

```python
>>> import io
>>> io.StringIO('aoeuidhtnsqjkxbmwpyfgcrl')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: initial_value must be unicode or None, not str
```
On 3.6 : 

```python
>>> import io
>>> io.StringIO('aoeuidhtnsqjkxbmwpyfgcrl')
<_io.StringIO object at 0x7f18b3704798>
```
This is why, I preferred to check which Python version is running. (Or I did not understand what did you say 😞 )